Readme.md

```diff
@@ -98,3 +98,24 @@ You did not rename `include/config.h.example`
 ### Failed to connect to ESP32: Timed out waiting for packet header
 
 You did not hold down BTN1(FLASH) and then tap the RESET button on the watch whilst platform.io was trying to connect.
+
+## OSW Emulator
+![emulator](./emulator.png)
+
+The OS itself can be executed as a regular program on your machine. This saves you time compiling for the watch and flashing the OS, every time you make a minor change - e.g. while developing the UI or a game, which not explicitly depend on the hardware of the watch.
+
+This also implies some limitations what you can do with the emulator, as we had to hack and reimplement some of the Arduino-specific libraries and their (conflicting) simplifications. This also means, that it maybe necessary to extend those extensions down the road as we (likely) missed that one specific function you try to use... :wink:
+
+### Build (cmake)
+The emulator can be build using the `CMakeLists.txt` file - you may need to install additional libraries to be able to use it.
+
+Here is a small example running on "Ubuntu 22.04 LTS":
+```bash
```

Comment:
Meh... Makes copy-paste harder. I think this is only relevant if we want to differentiate between root `#`and user `$`operations...

Reply:
IMHO, To specify that it is a command, it is recommended to have a special mark. Otherwise, beginners will confuse the command with the one that is not.